script.js

```diff
@@ -4,33 +4,211 @@ const logo = document.querySelector('.logo');
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    seeLiveLink: 'https://binhussen.github.io/portfolio/',
+    seeSourceLink: 'https://github.com/binhussen/portfolio',
+  },
+];
+
+const openModal = (i) => {
+  const projectTechs = projects[i].technologies.map(
+    (tec) => `<li class="tag">${tec}</li>`,
+  );
+
+  modal.classList.toggle('hidden');
+  wrapper.classList.toggle('blur');
+  overFlow.classList.toggle('scroll-off');
+
+  modal.innerHTML = `
+  <div class="modal-header">
+          <h1 class="primary-title modal-title">${projects[i].title}</h1>
+          <img src="assets/images/icons/Icon-Cancel.svg" alt="Cancel" class="cancel"/>
+        </div>
+        <ul class="tags">${projectTechs.join('')}
+        </ul>
+        <div class="works-stories">
+          <img
+            src=${projects[i].imageURL}
+            alt="Recent Works GYMFIT Multi-Post Stories image"
+            class="avatar modal-image"
+          />
+          <div class="container">
+            <p class="text-content works-content">
+              ${projects[i].description}
+            </p>
+            <div class="btns">
+              <a href=${
+  projects[i].seeLiveLink
+}><button type="button" class="btn">
+                See Live
+                <img
+                  class="btn-icon"
+                  src="./assets/images/icons/Icon-power.svg"
+                /></button></a>
+              </button>
+             <a href=${
+  projects[i].seeSourceLink
+}><button type="button" class="btn">
+                See Source
+                <img
+                  class="btn-icon"
+                  src="./assets/images/icons/github-white.svg"
+                />
+              </button></a>
+            </div>
+          </div>
+        </div>
+  `;
+};
+
+const showCards = (i) => {
+  const projectTechs = projects[i].technologies.map(
+    (tec) => `<li class="tag  card-tag">${tec}</li>`,
+  );
+
+  cards.innerHTML += `<div class="card">
+            <div class="card-content">
+              <h2 class="card-title">${projects[i].title}</h2>
+              <p class="text-content">${projects[i].content}
+              </p>
+              <ul class="tags">
+                ${projectTechs.join('')}
+              </ul>
+            </div>
+
+            <button type="button" class="btn mobile see-project">
+              See Project
+            </button>
+          </div>`;
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-plusplus
```

Comment:
- [ ] for good JS practices and clean code, please remove the code comments(eslint disable...) and fix any issue they solve as this might affect your future projects.

Reply:
- [ ] [REQUIRED] As the previous reviewer mentioned, please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1:

Reply:
Thank you @tufoinnkuo10 for your feedback, I will modify it.